site/content/en/references/kustomize/builtins/_index.md

```diff
@@ -768,7 +768,7 @@ helmChartInflationGenerator:
   releaseName: test
   namespace: testNamespace
   valuesFile: values.yaml
-```
```

Comment:
This is indeed worth fixing, but we want the opposite change: the text above should be in a code blocks, so it is the opening backticks that are missing. The "Example" subheading is also missing. Please take a look at other sections of this page and update to match.